templates/docs/auth.md

```diff
@@ -0,0 +1,749 @@
+h<% seoDescription("Third Party Authentication") %>
+<% seoKeywords(["buffalo", "go", "golang", "users", "password", "authentication"]) %>
+
+<%= h1("Local Authentication") %>
+
+In many use-cases, you'll need to implement user authentication in your apps.
+
+Buffalo had a native support for Auht until version `v0.9.4`. Since then, it g  was moved into it's own plugin, [https://github.com/gobuffalo/buffalo-auth](https://github.com/gobuffalo/buffalo-auth).
+
+<%= title("Installation") %>
+
+To install the `buffalo-auth` plugin, run the following command:
+
+```bash
+$ go get -u github.com/gobuffalo/buffalo-auth
+```
+
+<%= title("Generator") %>
+```bash
+$ buffalo g auth
+
+create  models/user.go
+create  models/user_test.go
+    run  goimports -w actions/actions_test.go actions/app.go actions/home.go actions/home_test.go actions/render.go grifts/db.go grifts/init.go main.go models/models.go models/models_test.go models/user.go models/user_test.go
+create  migrations/20180910062057_create_users.up.fizz
+create  migrations/20180910062057_create_users.down.fizz
+create  actions/auth.go
+create  actions/auth_test.go
+create  actions/users.go
+create  actions/users_test.go
+create  models/user_test.go
+create  actions/home_test.go
+create  templates/auth/new.html
+create  templates/index.html
+create  templates/users/new.html
+```
+
+<%= title("Example Usage") %>
+```go
```

Comment:
I think it's useful to have it here, first it allow you to see what is generated if you want to understand the code before generating it. It's also a good to have a point of reference for comparing a generated file, if  it's not working, maybe because of a corrupted plugin we can have a simple check. 

But I agree that I don't like to have them one after the other, maybe creating one placeholder with tabs could be a good solution, maybe @stanislas-m  can give us his opinion on this.

Reply:
Since we already quote the whole code on other examples, that seems fine to me.

Reply:
I've added code tabs to make it clearer.